tests/php/StoreApi/Utilities/OrderController.php

```diff
@@ -27,5 +27,6 @@ public function test_validate_selected_shipping_methods() {
 		$class->validate_selected_shipping_methods( true, array( 'free-shipping' ) );
 		$class->validate_selected_shipping_methods( false, array( 'free-shipping' ) );
 		$class->validate_selected_shipping_methods( true, null );
+		$this->assertTrue( true );
```

Comment:
Done. 

```
// The above methods throw Exception on error, but this is classed as a risky test because there are no 
// assertions. Assert true to work around this warning.
```